packages/contracts/src/utils.ts

```diff
@@ -244,8 +244,7 @@ const isValidStorageLayout = (storageLayout: any): boolean => {
     storageLayout !== null &&
     typeof storageLayout === 'object' &&
     Array.isArray(storageLayout.storage) &&
-    typeof storageLayout.types === 'object' &&
-    storageLayout.types !== null
+    typeof storageLayout.types === 'object'
```

Comment:
`null` has a type of `object`, so this returns `true` if `storageLayout.types` is `null`

Reply:
Damn